### PR TITLE
rename facades

### PR DESCRIPTION
**Pull request by KEggensperger**

### CODE REVIEW

File: examples/rf.py [hunk 1]
```diff
@@ -108,4 +106,4 @@ def rmse(y, y_pred):
     incumbent = smac.solver.incumbent
 
 inc_value = smac.get_tae_runner().run(incumbent, 1)[1]
-print("Optimized Value: %.2f" % (inc_value))
+print("Optimized Value: %.2f" % inc_value)
```
Comment (mlindauer) [inline examples/rf.py]: I want my paranthesis back T___T

File: smac/utils/io/cmd_reader.py [hunk 2]
```diff
@@ -136,6 +136,8 @@ class ReadPCSFileAction(Action):
     def __call__(self, parser: ArgumentParser, namespace: Namespace, values: list, option_string: str=None):
         fn = values
         if fn:
+            print(fn)
```
Comment (mlindauer) [inline smac/utils/io/cmd_reader.py]: please rm prints

**Comment by mlindauer** [comment]
Please check whether this PR still fails after merging with the new dev branch

**Comment by mlindauer** [comment]
the travis test on the examples fails with the message: `The job exceeded the maximum time limit for jobs, and has been terminated.`  How do we fix that? I would like to merge that PR asap such that I can use the new facades in SMACSMAC

**Comment by mlindauer** [comment]
The folder `doc/apidoc` is outdated such that not all files are rendered properly. Unfortunately, I don't know how to update the folder.

**Comment by KEggensperger** [comment]
Okay, sorry, I'll fix the remaining issue.

**Comment by mfeurer** [comment]
@shukon it seems that there's a problem with the docs, especially the apidocs directory. Is that generated automatically, or should we update that manually to reflect changes in the classes locations we're making?

**Comment by KEggensperger** [comment]
@mfeurer I already talked to André, the apidoc folder can be generated automatically. However, by looking at `create_doc.sh` I would guess it is not created automatically with every commit. I would expect adding ```make buildapi``` in line 23 should fix this issue. What do you think?

**Comment by mfeurer** [comment]
I think you're right and I think it would be great to have the apidoc directory be generated automatically during every build.

**Pull request merged**